primordialsoup/static/css/primordialsoup.css

```diff
@@ -687,6 +677,39 @@ hr.map-event-detail-top {
     line-height: 35px;
 }
 
+/* leaflet map 'view all markers button */
+div.leaflet-bar.leaflet-control a.leaflet-view-all {
+    width: 30px;
+}
+
+div.leaflet-bar.leaflet-control a.leaflet-view-all a {
+    position: relative; 
+    background: transparent;
+    font-size: 24px;
+}
+
+@media only screen and (max-width: 435px) {
```

Comment:
Where did you get 435px from? I think we used 599px for other media queries for mobile, but please double check. Lets be consistent about this 🥇

Reply:
The number 435 taken from div.map-circle (line 244). I can try switching to 599px and see if it still works?